daprdocs/content/en/operations/components/setup-bindings/supported-bindings/http.md

```diff
@@ -19,19 +19,87 @@ spec:
   metadata:
   - name: url
     value: http://something.com
-  - name: method
-    value: GET
 ```
 
 - `url` is the HTTP url to invoke.
-- `method` is the HTTP verb to use for the request.
 
 ## Output Binding Supported Operations
 
-* create
+### Retrieving data
+
+To retrieve data from the HTTP endpoint, invoke the HTTP binding with a `GET` method and the following JSON body:
+
+```json
+{
+  "operation": "get"
+}
+```
+
+Optionally, a path can be specified to interact with resource URIs:
+
+```json
+{
+  "operation": "get",
+  "metadata": {
+    "path": "/things/1234"
+  }
+}
+```
+
+#### Response
+
+The response body will contain the data returned by the HTTP endpoint.  The `data` field contains the HTTP response body as a byte slice (Base64 encoded via curl). The `metadata` field contains:
+
+* `statusCode` for the [HTTP status code](https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html)
+* `status` for the status description (e.g. 200 OK, 201 Created, etc.)
+* Values for all the HTTP response headers. If multiple values for te same key exist, they are delimited by `, `.
+
+#### Example:
+
+> We escape since ' is not supported on Windows
```

Comment:
This gets concatonated into a single line when rendered. Use an alert shortcode instead:

{{% alert title="Title of note here" color="primary" %}}
Place content here
{{% /alert %}}